docs/usage/help.rst.inc

```diff
@@ -96,6 +96,9 @@ selector prefix is also supported for patterns loaded from a file. Due to
 whitespace removal paths with whitespace at the beginning or end can only be
 excluded using regular expressions.
 
+To test your exclusion patterns without performing an actual backup you can
+run ``borg create --list --dry-run ...``.
+
```

Comment:
"IMPORTANT: this file is auto-generated from borg's built-in help, do not edit!" (1st line of this file).

You need to locate the same text in `src/borg/archiver.py` and change it there only, all other stuff will be auto-created from that at release time.

Reply:
Ack, PR updated.